### PR TITLE
Feature/22966 journey api

### PR DESCRIPTION
**Pull request by nnorbert**
Journey API client added to the SDK.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "[typescript]": {
-    "editor.defaultFormatter": "biomejs.biome"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```
Comment (matteolc) [inline .vscode/settings.json:3]: Why this change?

**Review by matteolc — COMMENTED** [review]
Can we add some tests please?

File: .vscode/settings.json (hunk 1 above)
Comment (nnorbert) [reply .vscode/settings.json:3]: I don't know, maybe this file should not even be pushed but ignored. It is VSCode related setting.

Comment (nnorbert) [reply .vscode/settings.json]: Reverted it!

**Review by matteolc — APPROVED** [review]

**Review by SamSalvatico — APPROVED** [review]
Thank you @nnorbert !
